indico/MaKaC/PDFinterface/conference.py

```diff
@@ -430,7 +431,7 @@ def __init__(self, conf, aw, contribs=None, tz=None, sort_by=""):
             'fields': [f for f in event.contribution_fields if f.is_active],
             'sorted_by': sort_by,
             'aw': aw,
-            'boa_text': conf.getBOAConfig().getText()
+            'boa_text': boa_settings.get(conf.as_event, 'extra_text')
```

Comment:
I would use the opportunity to change the method to accept `event` instead of `conf`, then use just `event` here and `event.as_legacy` in places where you do need the old conference object.